### ISSUE TITLE
'reset state' button seems not to work - not important as far as I can tell

### ISSUE THREAD

**Issue by Lew-Rockwell-Fan** [yacy/yacy_search_server#145]
On this page:

http://localhost:8090/Performance_p.html

in the 'RAM' section,
to the right of
'Memory state: exhausted' & an i icon
there is a button 'reset state'.

Not sure what it's supposed to do but clicking it gives me:

```
HTTP ERROR 405

Problem accessing /Performance_p.html. Reason:

    HTTP POST method is the only one authorized.


YaCy 1.921 - powered by Jetty - 
```

which doesn't seem useful.

This isn't a problem for me, AFAIK, but I figure anything that seems wrong is worth reporting. Honestly, I'm not running yacy because it is really useful yet, but because I hope to understand it well enough to contribute bug reports that might be helpful in a small way.

**Comment by luccioman** [comment]
Thanks @Lew-Rockwell-Fan for your report. This is now fixed by commit c5c3cc1.
A few more explanations : the "reset state" button is only useful to manually reset the monitoring state of RAM memory, when the state is 'exhausted'. That state detected as 'exhausted' when some operations could not be performed because they required more memory than available without going below the user configurable "Minimum required" value (default to 50 MiB). 
By manually resetting, it lets you now if memory soon become again exhausted or no.

The message you got saying "HTTP POST method is the only one authorized" should effectively not have been displayed under normal operation. It is just a consequence of some security refactoring done earlier, allowing as stated only HTTP POST method for operations that modify the server state (HTTP GET method semantics imply it should be limited to read-only operations).

**Comment by luccioman** [comment]
Fix embedded in developer release [1.921.9513](https://github.com/luccioman/yacy_search_server/releases/tag/Release_1.921.9513-dev)
